document/CoreFormat.mdk

```diff
@@ -528,7 +586,7 @@ The `#pragma` warning construct in C# is such a syntactic construct.
 
 ### `fixes` property { #result-fixes }
 
-A `result` object may contain a property names `fixes`, whose value is a JSON array of `fix` objects (&sect;[#fix]).
+A `result` object may contain a property names `fixes`, whose value is a JSON array of zero or more `fix` objects (&sect;[#fix]).
```

Comment:
sorry, I know this keeps coming up, this language implies that it is ok to add an empty array, with no fixes, to the log file. how do we handle this elsewhere? i.e., do we permit other arrays both to be emitted entirely or to exist as an empty array to indicate there is no data associated with the property? do we permit this as a convenience for results production? it is arguably a slight inconvenience in results consumption. I'm not sure I have a strong feeling on this topic, trying to recall what we've said about it in the past and to ensure that we're consistent across the standard.